src/app/shared/pipes/userCaption.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { User } from '../../modules/group/http-services/get-user.service';
+
+@Pipe({ name: 'userCaption', pure: true })
+export class UserCaptionPipe implements PipeTransform {
+  transform(user: User): any {
+    return user.firstName && user.lastName ? `${ user.firstName } ${ user.lastName } (${ user.login })` : user.login;
```

Comment:
👍 👍 
Actually, do `user.firstName || user.lastName` if we should display it is only one is given.
So you have to handle the double spaces if one of the two is empty/null/undefined.

Reply:
Fixed